resources/html/index.html

```diff
@@ -5,27 +5,32 @@
     <title>Dashboard</title>
     <meta name="description" content="Dashboard for github pull-requests notifier">
     <meta name="author" content="wentwrong">
-    <script src="https://cdn.jsdelivr.net/npm/systemjs/dist/system.js"></script>
-    <script src="https://cdn.jsdelivr.net/npm/systemjs/dist/extras/named-register.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/systemjs@6.2.3/dist/system.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/systemjs@6.2.3/dist/extras/named-register.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/systemjs@6.2.3/dist/extras/amd.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/systemjs@6.2.3/dist/extras/global.js"></script>
     <script type="systemjs-importmap">
-        {
-          "imports": {
-            "dedent": "https://www.unpkg.com/dedent@0.7.0/dist/dedent.js",
-            "debug": "https://unpkg.com/debug@4.1.1/dist/debug.js"
-          }
+      {
+        "imports": {
+          "debug": "https://unpkg.com/debug@4.1.1/dist/debug.js"
         }
-    </script>
+      }
+  </script>
     <script src="bundle.js"></script>
 </head>
 
 <body>
   <div id="root">
     Loading...
   </div>
-
   <script>
     (async () => {
+      const { default: globalClientErrorHandlers } = await System.import('shared/globalErrorHandlers');
```

Comment:
We have to use a separate bundle for error handlers. Otherwise, a syntax error in the bundle will prevent error handlers from initialization. However, this PR is already too big. We can do it as a separate task if you want.